src/js/components/DataSearch/DataSearch.js

```diff
@@ -128,7 +128,7 @@ export const DataSearch = ({
       kind={theme.data.button?.kind}
       icon={<Search />}
       dropProps={dropProps}
-      dropContent={<Box pad="small">{content}</Box>}
+      dropContent={<Box {...theme.data?.drop}>{content}</Box>}
```

Comment:
@jcfilben How do you feel about spreading all box props here vs only supporting pad to start?

I think I'd done this one, but just commenting now based on our discussion with Brittany.

Reply:
oh yeah I forgot I left this one I think we only wanted `pad`?